collection-scripts/gather_monitoring

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+
+function collect() {
+  local resources target 
+  resources=()
+  target="$1"
+
+  mkdir -p "${BASE_COLLECTION_PATH}/${target}"
+  for i in $(/usr/bin/oc get "${target}.monitoring.coreos.com" --all-namespaces | grep "openshift-" | awk '{ print $1 }')
```

Comment:
Are we sure we have awk in the container? 
We should consider using a template for this.

Reply:
I can rewrite it to not use `awk` if needed.

Reply:
used `cut` instead.

Reply:
replacing awk with cut doesn't remove the dependency on an external tool, not does it say in alignment with other scripts, where we use templates, see https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_network_logs#L11